code/ATMOSPHERICS/atmospherics.dm

```diff
@@ -43,6 +43,9 @@ Pipelines + Other Objects -> Pipe network
 	plane = ABOVE_TURF_PLANE
 	layer = PIPE_LAYER
 	var/piping_layer = PIPING_LAYER_DEFAULT //used in multi-pipe-on-tile - pipes only connect if they're on the same pipe layer
+	//We brought this down to the atmospherics level. You don't need to use it for most pipes, but it's used for some things like machineries!
+	var/obj/machinery/atmospherics/node1
```

Comment:
@Exxion there was a good reason for this not to be here, right?

Reply:
Well, not everything has two nodes

Reply:
any opinion on this?

Reply:
Well, I don't see any particular reason to change it from how it was. I'd recommend changing it back just for consistency, but I don't *think* it would break anything either way.